### PR TITLE
NGGW-30 implement 'vertical margin' block plugin in netgen layouts

### PR DESCRIPTION
**Pull request by kdraganic**
Implementation of 'Vertical margin' block plugin in Netgen Layouts

Related PR https://github.com/netgen/site-bundle/pull/28

### CODE REVIEW

**Review by iherak — COMMENTED** [review]
Is there a related PR here? If there is, please link to it in the description.

**Comment by emodric** [comment]
There is, and I told @kdraganic to link it ;) :P

File: src/AppBundle/Resources/sass/blocks/_block.scss [hunk 1]
```diff
@@ -18,6 +18,38 @@
...
+            &.whitespace-top-#{$keyMap} {
+                  padding-top: $valueMap;
+            }
+            &.whitespace-bottom-#{$keyMap} {
+                  padding-bottom: $valueMap;
+            }
+        }
+    }
+
+    @each $keyMap, $valueMap in $sizes {
```
Comment (emodric) [inline src/AppBundle/Resources/sass/blocks/_block.scss:33]: This should really use separate variables for margin sizes, so we can change one, without risking change to the other.

**Comment by Ljudevit** [comment]
Not needed anymore because of components introduction.

**Pull request closed without merging**